test/functional/ui/multigrid_spec.lua

```diff
@@ -2599,4 +2599,256 @@ describe('ext_multigrid', function()
     ]])
     eq(1, api.nvim_get_option_value('cmdheight', {}))
   end)
+
+  describe('centered cursorline', function()
+    before_each(function()
+      -- These settings simulate / achieve / force ... #30576
+      command('set cursorline scrolloff=9999')
```

Comment:
just to be clear: I was hoping you would fill in the rest of the comment :)

Reply:
Ok, I will check that again

Reply:
Ok, it should be better now :)